internal/setup/webhooks.go

```diff
@@ -18,24 +18,23 @@ import (
 )
 
 const (
-	serviceName     = "hnc-webhook-service"
-	vwhName         = "hnc-validating-webhook-configuration"
-	mwhName         = "hnc-mutating-webhook-configuration"
-	caName          = "hnc-ca"
-	caOrganization  = "hnc"
-	secretNamespace = "hnc-system"
-	secretName      = "hnc-webhook-server-cert"
-	certDir         = "/tmp/k8s-webhook-server/serving-certs"
+	serviceName    = "hnc-webhook-service"
+	vwhName        = "hnc-validating-webhook-configuration"
+	mwhName        = "hnc-mutating-webhook-configuration"
+	caName         = "hnc-ca"
+	caOrganization = "hnc"
+	secretName     = "hnc-webhook-server-cert"
+	certDir        = "/tmp/k8s-webhook-server/serving-certs"
 )
 
-// DNSName is <service name>.<namespace>.svc
-var dnsName = fmt.Sprintf("%s.%s.svc", serviceName, secretNamespace)
-
 // ManageCerts creates all certs for webhooks. This function is called from main.go.
-func ManageCerts(mgr ctrl.Manager, setupFinished chan struct{}, restartOnSecretRefresh bool) error {
+func ManageCerts(mgr ctrl.Manager, setupFinished chan struct{}, restartOnSecretRefresh bool, hncNamespace string) error {
```

Comment:
If you add a `config.GetHNCNamespace()` function, you can (and should) remove `hncNamespace` from the param list here.

Reply:
I thought about it yesterday hh, about provide a common way to get hnc namespace. OK, I'll modify it.